modules/installation-vsphere-infrastructure.adoc

```diff
@@ -52,34 +52,34 @@ You must install the {product-title} cluster on a VMware vSphere version 7 insta
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
-|VM hardware version | 15 or later
+|VM hardware version | 17 or later
 |vSphere ESXi hosts | 7.0.2 or later
 |vCenter host   | 7.0.2 or later
 |===
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere version 7.0.1 or earlier is now deprecated. These versions are still fully supported, but version 4.11 of {product-title} requires vSphere virtual hardware version 15 or later. Hardware version 15 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
+Installing a cluster on VMware vSphere version 7.0.1 or earlier is now deprecated. These versions are still fully supported, but version 4.12 of {product-title} requires vSphere virtual hardware version 17 or later. Hardware version 17 is now the default for vSphere virtual machines in {product-title}. To update the hardware version for your vSphere nodes, see the "Updating hardware on nodes running in vSphere" article.
```

Comment:
These versions -> this version

Wrt the HW "version 17 or later", 7.0.2 comes with HW 19 (https://kb.vmware.com/s/article/1003746). Shouldn't requiring 7.0.2 go along with HW19? Or could we be on 7.0.2 but install a new cluster with HW17 at all?

Reply:
Isn't HW version 19 the default in 4.12?